Anabi.DataAccess.Ef/AnabiContext.cs

```diff
@@ -40,6 +40,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
             (new AssetsStorageSpacesConfig()).SetupEntity(modelBuilder);
             base.OnModelCreating(modelBuilder);
+            (new RecoveryBeneficiaryConfig()).SetupEntity(modelBuilder);
+            base.OnModelCreating(modelBuilder);
```

Comment:
OnModelCreating is already called on the line above. I don't think you should call it twice